embedded-pubsub/src/main/java/com/playtika/test/pubsub/PubsubProperties.java

```diff
@@ -0,0 +1,20 @@
+package com.playtika.test.pubsub;
+
+import com.playtika.test.common.properties.CommonContainerProperties;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.Collection;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ConfigurationProperties("embedded.pubsub")
+public class PubsubProperties extends CommonContainerProperties {
+    public static final String BEAN_NAME_EMBEDDED_PUBSUB = "embeddedPubsub";
+    private String dockerImage = "google/cloud-sdk:latest";
```

Comment:
usage of 'latest' is anti-pattern - released lib might be broken due to updates which you do not expect